tests/countries/test_anguilla.py

```
@@ -0,0 +1,322 @@
+            "2018-05-21",
+            "2019-06-10",
+            "2020-06-01",
+            "2021-05-24",
+            "2022-06-06",
+            "2023-05-29",
+            "2024-05-20",
+            "2025-06-09",
+        )
+        self.assertHolidayName(name, range(2011, 2050))
+
+    def test_celebration_of_the_birthday_of_her_majesty_the_queen(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)